src/Psalm/Internal/Stubs/CoreGenericFunctions.phpstub

```diff
@@ -249,11 +249,12 @@ function uksort(array &$arr, callable $callback): bool
 }
 
 /**
+ * @psalm-template K of array-key
```

Comment:
:thinking: maybe I should remove the ` of array-key` part?

Reply:
You may do that, however it really shouldn't do any harm. After all, array keys could only be `array-key`. Come to think of it, it may even be useful to catch situations when Psalm wrongly infers key types, like here: https://psalm.dev/r/5c1dae6bd1